### PR TITLE
Add onboarding capture and scanning screens

### PR DESCRIPTION
**Pull request by bogiii**
#### Changes proposed in this Pull Request

Changes introduce additional Capture and Scanning screens.

Those screens are live at the following routes:
- `importer/capture`
- `importer/scanning`

The initial step is `capture`, then `scanning`, and depending on the scanning result, the user will be redirected to the `list` screen. If the user refreshes the screen, he will be redirected to the initial step.

#### Testing instructions
The important thing to share is that we are still providing UI without business logic and everything is hidden behind the feature flag.

Manual testing:
- go to `/start/importer/`
- enter any valid URL and hit the return key
- `scanning` screen will be shown with the spinner placeholder (spinner will be replaced in future improvements)
- after a few second, the `list` screen will be presented

#### Screenshots
Screenshot 2021-10-22 at 12 43 57

Screenshot 2021-10-22 at 12 44 37

Related to
- [Figma design](https://www.figma.com/file/TAK5FpwTvfbENJDfHc7Aq1/Site-Importing?node-id=1930%3A21706)

### CODE REVIEW

File: client/signup/steps/import/capture/index.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+	return (
+		<div className="import-layout__center">
+			<div className="capture__content">
+				<input
+					className="capture__input"
+					autoComplete="off"
+					autoCorrect="off"
+					spellCheck="false"
+					placeholder={ __( 'Enter your site address' ) }
```
Comment (a8ci18n) [inline client/signup/steps/import/capture/index.tsx:48]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Enter your new site address' )` *ES Score: 7*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/gutenboarding-import-capture)

**Review by jeroenpf — COMMENTED** [review]
@bogiii  It looks good to me. I am not very familiar with the flow logic so I had to dig into that a bit.

I have one note: when i am on the `/list` page and click on the back button to go back to the capture screen, it seems to be a bit buggy. I have to click twice to go back, but it brings me back to the scanning screen. 

Perhaps the scanning step doesn't have to be a separate page and we can just add logic on the /list page that shows the spinner if the analyzer is still busy?

**Comment by bogiii** [comment]
> Perhaps the scanning step doesn't have to be a separate page and we can just add logic on the /list page that shows the spinner if the analyzer is still busy?

Yes, you are right. We probably don't need a dedicated scanning step, and it should be part of the capture step.

The idea was to provide UI, but we should move it to the capture screen in future changes.

@jeroenpf, if you agree, I would merge it so we can keep moving with this effort?

**Comment by jeroenpf** [comment]
> @jeroenpf, if you agree, I would merge it so we can keep moving with this effort?

Yes, lets do it :-)

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/6901444

Thank you @bogiii for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/6901444).

**Pull request merged**